src/components/Table/Table.story.jsx

```diff
@@ -9,7 +9,15 @@ import Add from '@carbon/icons-react/lib/add/16';
 import Edit from '@carbon/icons-react/lib/edit/16';
 import { spacing03 } from '@carbon/layout';
 import { Add20, TrashCan16, SettingsAdjust16 as SettingsAdjust } from '@carbon/icons-react';
-import { Tooltip, TextInput, Checkbox, ToastNotification, Button } from 'carbon-components-react';
+import {
+  Tooltip,
+  TextInput,
+  Checkbox,
+  ToastNotification,
+  Button,
+  FormGroup,
+  Form,
+} from 'carbon-components-react';
```

Comment:
I know it's just a story file, but would prefer to use as few direct imports from `carbon-components-react` as possible and instead use our entrypoint.

```suggestion
} from '../../index';
```

Reply:
I agree, will change this.